sdcm/utils/sstable/sstable_utils.py

```diff
@@ -199,12 +190,29 @@ def get_table_repair_date(self) -> str | None:
     def get_table_repair_date_and_delta_minutes(self) -> (datetime.datetime, int):
 
         table_repair_date = datetime.datetime.strptime(self.get_table_repair_date(), '%Y-%m-%d %H:%M:%S')
+        self.log.debug('Found table-repair-date: %s', table_repair_date)
         now = datetime.datetime.now()
-        delta_repair_date_minutes = ((now - table_repair_date).seconds - self.propagation_delay_in_seconds) // 60
-        self.log.debug('Found table-repair-date: %s, Ended %s minutes ago',
-                       table_repair_date, delta_repair_date_minutes)
+        gross_delta_sec = (now - table_repair_date).seconds
+        self.log.debug('The full delta seconds from now is: %s', gross_delta_sec)
+        propagation_delay_subtraction_delta_sec = gross_delta_sec - self.propagation_delay_in_seconds
+        self.log.debug('The delta seconds from now, subtracting propagation delay, is: %s',
```

Comment:
Too many debug messages, I think we need only the final one. All others are effectively spam

Reply:
fixed